lib/anubis.go

```diff
@@ -310,19 +312,29 @@ func (s *Server) PassChallenge(w http.ResponseWriter, r *http.Request) {
 
 	challenge := s.challengeFor(r, rule.Challenge.Difficulty)
 
+	if _, err := r.Cookie(anubis.TestCookieName); err == http.ErrNoCookie {
+		s.ClearCookie(w, s.cookieName)
+		s.ClearCookie(w, anubis.TestCookieName)
+		lg.Warn("user has cookies disabled, this is not an anubis bug")
+		s.respondWithError(w, r, "Your browser is configured to disable cookies. Anubis requires cookies for the legitimate interest of making sure you are a valid client. Please enable cookies for this domain")
```

Comment:
nit: missing final `.`

Reply:
respondWithError adds it

Reply:
Maybe this is confusing behavior that should be removed lol